Macaw.podspec

```diff
@@ -10,21 +10,21 @@ Pod::Spec.new do |s|
   s.name             = "Macaw"
   s.version          = "0.9.5"
   s.summary          = "Powerful and easy-to-use vector graphics library with SVG support written in Swift."
-
+  
   s.homepage         = 'https://github.com/exyte/Macaw.git'
   s.license          = 'MIT'
   s.author           = { 'Exyte' => 'info@exyte.com' }
   s.source           = { :git => 'https://github.com/exyte/Macaw.git', :tag => s.version.to_s }
   s.social_media_url = 'http://exyte.com'
-
+  
   s.ios.deployment_target = "9.0"
   s.osx.deployment_target = "10.12"
   s.requires_arc = true
   s.swift_version = "5.0"
-
+  
   s.source_files = [
-     'Source/**/*.swift'
+    'Macaw/Sources/Macaw/**/*.swift'
```

Comment:
Why are you decide to move sources?

Reply:
That is the standard file structure for an SPM package